packages/plugins/Web3Profile/src/SNSAdaptor/index.tsx

```diff
@@ -72,6 +72,9 @@ const sns: Plugin.SNSAdaptor.Definition = {
             ID: PluginId.Web3Profile,
             label: 'Web3Profile',
             priority: 2,
+            UI: {
+                TabContent: PublicWalletSetting,
```

Comment:
The same `TabContent` has been exported in the `Tips` plugin.

Reply:
Yep, I also added in `Tips ` with `PublicWalletSetting `.